telosnetwork/telos-wallet
# #235 | Tooltip component

# Fixes #235 

## Description
PR adds a tooltip component with styling.

## Test scenarios
- go to https://deploy-preview-251--wallet-staging.netlify.app/evm/wallet
- make screen mobile-sized so that numbers are truncated
- click tooltip icon on different rows
    - tooltips should appear and look good

## Checklist:
